upstream-community-operators/camel-k/camel-k.v0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,273 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    repository: https://github.com/apache/camel-k
+    containerImage: docker.io/apache/camel-k:0.3.3
+    support: Camel
+    createdAt: 2019-05-06T16:45:00Z
+    certified: "false"
+    description: |-
+      Apache Camel K (a.k.a. Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+    alm-examples: |-
+      [{
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "IntegrationPlatform",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "build": {
+            "buildStrategy": "pod"
+          },
+          "resources": {
+            "contexts": [
+              "jvm"
+            ]
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "Integration",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "source": {
+            "content": "// Add example Java code to create Integration",
+            "name": "Example.java"
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
```

Comment:
Is it expected that the `IntegrationContext` CR have no `spec` defined?
If this and other CRs are expected to be used behind the scenes (not user or admin facing), then please let us know.

Reply:
`IntegrationContext`, `Build` and `CamelCatalog` are CRs that should not be _user-facing_, and are internal to the operator. Ideally, there should be a way to hide then from the UI, which explain why I did not create a `spec` section. Let me know what's the best strategy to handle this kind of CRs.

Reply:
Sounds good. At the moment there is no way to hide CRs from the UI since CRDs are public. No changes required here :)

Reply:
I am sorry, but I am not writing the camel operator, you may be commenting on the wrong pull request